src/systemtask/SystemTask.cpp

```diff
@@ -56,7 +56,7 @@ SystemTask::SystemTask(Drivers::SpiMaster& spi,
                        Controllers::Ble& bleController,
                        Controllers::DateTime& dateTimeController,
                        Controllers::TimerController& timerController,
-                       Controllers::AlarmController& alarmController,
+                       Pinetime::Controllers::AlarmController& alarmController,
                        Drivers::Watchdog& watchdog,
                        Pinetime::Controllers::NotificationManager& notificationManager,
                        Pinetime::Controllers::MotorController& motorController,
```

Comment:
You need to pass an instance of `Pinetime::Controllers::AlertController` around here. It should probably look like:
`Pinetime::Controllers::AlertController& alertController,`

Compare with other controllers like the MotorController.